src/hardware/gus.cpp

```diff
@@ -950,12 +970,14 @@ void Gus::PrintStats()
 	LOG_MSG("GUS: Peak amplitude reached %.0f%% of max",
 	        100 * static_cast<double>(peak_ratio));
 
-	// Make a suggestion if the peak volume was well below 3 dB
-	if (peak_ratio < 0.6f) {
+	// Make a suggestion if the peak volume was well below 3 dB, but without
+	// the influence of the mixer's scalar because the user might have
+	// deliberately set it low.
+	if (peak_ratio / scalar < 0.6f) {
 		const auto multiplier = static_cast<uint16_t>(
-		        100 * mixer_scalar / peak_ratio);
+		        100 * scalar / peak_ratio);
 		LOG_MSG("GUS: If it should be louder, %s %u",
-		        fabs(mixer_scalar - 1.0f) > 0.01f ? "adjust mixer gus to"
+		        fabs(scalar - 1.0f) > 0.01f ? "adjust mixer gus to"
```

Comment:
Let's store the result of this `fabs` call in a const variable for readability reasons.

Reply:
Also moved in a later commit and doesn't exist - however, was able to do something similar in the soft_limit class where we use fabs() in the peak calculation.  I think it similarly improves readability.